flatpak/com.matthiasnehlsen.lotti.yml

```
@@ -15,30 +15,36 @@
-              install -D assets/icon/app_icon_1024.png /app/share/icons/hicolor/${size}x${size}/apps/com.matthiasnehlsen.lotti.png
+              install -D app_icon_1024.png /app/share/icons/hicolor/${size}x${size}/apps/com.matthiasnehlsen.lotti.png
             done
         sources:
           - type: file
-            path: flatpak/com.matthiasnehlsen.lotti.desktop
+            path: com.matthiasnehlsen.lotti.desktop
           - type: file
-            path: flatpak/com.matthiasnehlsen.lotti.generated.metainfo.xml 
+            path: com.matthiasnehlsen.lotti.generated.metainfo.xml
+          - type: file
+            path: ../assets/icon/app_icon_1024.png 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)